lib/storage-service.js

```diff
@@ -100,7 +112,11 @@ class StorageService {
     return this.defaultToken.initialize().then(() => {
       // If the LeaseManager is already provisioned, return it
       if (this._managers.has(secretID)) {
-        return this._managers.get(secretID).initialize();
+        const manager = this._managers.get(secretID);
+
+        StorageService.logManagerEvent('DEBUG', 'Retrieved LeaseManager.', manager);
```

Comment:
Do we want to log `manager` here? That will include `manager.data` which is the secret. Maybe use `StorageService.logManagerEvent` instead?